src/components/RecordDetail/Relations.astro

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
We have an existing `AccordionItemsList` component that might be useful here.

Reply:
I don't think the accordion would match the [designs](https://www.figma.com/design/5AWa1c3pJeVXwEeVPbRqZj/Place-Components-2.0?node-id=713-4649&m=dev). I'm also not sure that being able to collapse/expand the lists of relations would be desired on a detail page.